package.json

```diff
@@ -39,5 +43,15 @@
     "karma-junit-reporter": "^1.2.0",
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-requirejs": "^1.1.0"
-  }
+  },
+  "keywords": [
```

Comment:
Nice catch with the "browser" keywords. Let's add "browser" to GitHub and keep the two lists identical. Imagine trying to decide what keywords to use where when the two are different. It's much more difficult to maintain.